subxt/src/metadata/decode_encode_traits.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019-2022 Parity Technologies (UK) Ltd.
```

Comment:
nit: Substrate has decided to remove the copyright years, I guess we could adopt the same style in subxt since we'll no longer care about incrementing years :D